lib/omniauth/strategies/bolt.rb

```diff
@@ -62,6 +74,9 @@ def callback_phase
         response = JSON.parse(res.body)
         @access_token = response['access_token']
         @user_uid = response['id_token']
+        session[:expiration_time] = Time.now + response['expires_in']
+        session[:refresh_token] = response['refresh_token']
+        session[:refresh_token_scope] = response['refresh_token_scope']
```

Comment:
Can you add the `bolt_` prefix to these fields, and the `authorization_code` as well?

Reply:
reworked logic in https://github.com/nebulab/omniauth-bolt/pull/2/commits/9b3154e439f4f511d1c1335517c5b9ba666567ab